KsApi/models/User.swift

```diff
@@ -12,6 +12,7 @@ public struct User {
   public let name: String
   public let newsletters: NewsletterSubscriptions
   public let notifications: Notifications
+  public let optedOutOfRecommendations: Bool?
```

Comment:
It might be better to have this be non-optional with a default value of `false`. That way you can always assume the value is non-nil.